Tests/BootCamp.Chapter.Tests/ProgramTests.cs

```diff
@@ -17,17 +17,27 @@ public class ProgramTests : IDisposable
         [InlineData("nonExisting.csv")]
         public void Main_When_Transactions_File_Empty_Or_Not_Found_Throws_NoTransactionsException(string input)
         {
-            Action action = () => Program.Main(new[] { input });
+            const string cmd = "time";
+
+            Action action = () => Program.Main(new[] { input, cmd, OutputFile });
 
             action.Should().Throw<NoTransactionsFoundException>();
         }
 
+        [Fact]
+        public void Main_When_Too_Few_Arguments_Throws_InvalidCommandException()
+        {
+            Action action = () => Program.Main(new[] { ValidTransactionsFile });
+
+            action.Should().Throw<InvalidCommandException>();
+        }
+
         [Fact]
         public void Main_When_Command_Invalid_Throws_InvalidCommandException()
```

Comment:
Modified to test an otherwise valid set of arguments, but with an invalid command.